presto-verifier/src/test/java/com/facebook/presto/verifier/checksum/TestChecksumValidator.java

```diff
@@ -67,7 +69,9 @@
     private static final Column INT_ARRAY_COLUMN = new Column("int_array", ARRAY, new ArrayType(INTEGER));
     private static final Column ROW_ARRAY_COLUMN = new Column("row_array", ARRAY, typeRegistry.getType(parseTypeSignature("array(row(a int,b varchar))")));
     private static final Column MAP_ARRAY_COLUMN = new Column("map_array", ARRAY, typeRegistry.getType(parseTypeSignature("array(map(int,varchar))")));
-    private static final Column ROW_COLUMN = new Column("row", ROW, typeRegistry.getType(parseTypeSignature("row(a int,double,array(row(a int,b varchar)),b varchar)")));
+    private static final Column ROW_COLUMN = new Column("row", ROW, typeRegistry.getType(parseTypeSignature("row(a int,varchar,double)")));
+    private static final Column ROW_WITH_ARRAY_COLUMN = new Column("row_with_array", ROW, typeRegistry.getType(parseTypeSignature("row(a array(int))")));
+    private static final Column ROW_WITH_ROW_COLUMN = new Column("row_with_row", ROW, typeRegistry.getType(parseTypeSignature("row(row(double))")));
```

Comment:
Instead of having 3 columns, let's add the array field and the row field in to `ROW_COLUMN`.
Also, reduce the # of columns without field name so the test can be more readable. 
For example: A suitable test could look like:
```Java
parseTypeSignature("row(i int, varchar, d double, a array(int), r row(double, b bigint))"
```